src/Ocelot.Provider.Polly/PollyQoSProviderBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Ocelot.Configuration;
+using System.Net;
+
+namespace Ocelot.Provider.Polly;
+
+public class PollyQoSProviderBase
+{
+    protected static readonly HashSet<HttpStatusCode> ServerErrorCodes = new()
```

Comment:
Collection initialization can be simplified, removing new(), using collection expression
```suggestion
    protected static readonly HashSet<HttpStatusCode> ServerErrorCodes = [...]
```

Reply:
minor issue